lib/controllers/main_info.dart

```diff
@@ -51,6 +51,13 @@ class MainInfo {
 
   String get outputPath => configuration.outputPath;
 
+  Map<String, dynamic> amplitudMap = {
+    'eventProperties': {
+      'tags': {},
+      'eggs': [],
```

Comment:
I don't think we should refer to `eggs` anymore

Reply:
Too bad, but you are right!